web/src/hashchange.js

```diff
@@ -199,6 +199,15 @@ function do_hashchange_normal(from_reload) {
             inbox_ui.show();
             break;
         case "#all_messages":
+            // "#all_messages" was deprecated and replaced by
+            // "#feed" in 2024 so if a user goes to "#all_messages"
+            // (through saved boookmark or any such scenario) they'll
```

Comment:
`boookmark` is not spelled correctly. I highly recommend using a tool like `codespell` to spell check your code, but also taking more time to be careful in writing -- it's one of the most important parts of a project like this.